src/layout/header/Header.tsx

```diff
@@ -64,8 +64,8 @@ export const Header = () => {
               <IconButton onClick={handleClick}>
                 <MenuIcon />
               </IconButton>
+              <MobileMenu anchorEl={anchorEl} onClose={() => setAnchorEl(null)} />
             </StyledBurgerMenu>
-            <MobileMenu anchorEl={anchorEl} onClose={() => setAnchorEl(null)} />
```

Comment:
hvilken forskjell gjør dette egentlig? `StyledBurgerMenu` er jo en div?

Reply:
og hvis det skal være sånn, så kan `<>` og `</>` fjernes

Reply:
Når det var utenfor ble det nå en ekstra div for popoveren i headeren som fucka opp aligning. (tidligere var den utenfor, i `<body>`)
fjerner `<>`